.github/dependabot.yml

```diff
@@ -16,7 +16,7 @@ updates:
   - package-ecosystem: "npm"
     directory: "/frontend"
     schedule:
-      interval: "weekly"
```

Comment:
Graph display menu has misalignment of text:

Before:
![Screenshot from 2022-05-03 12-53-21](https://user-images.githubusercontent.com/6226732/166501750-52483e60-235d-4db3-8df8-2cffedfd8485.png)

After:
![Screenshot from 2022-05-03 12-53-07](https://user-images.githubusercontent.com/6226732/166501766-880d28fd-3583-4323-bc9d-506dc9cb5f44.png)